### PR TITLE
DCOS-12858: Update download endpoint for logs

### PR DESCRIPTION
**Pull request by mlunoe**
Resolves DCOS-12761 as well.

This PR updates the download endpoint for the logs:

Please review using a cluster from this PR: https://github.com/dcos/dcos/pull/1122

### CODE REVIEW

**Comment by jfurrow** [comment]
When I click on the `Logs` tab for apps, I get the following fatal error:

File: plugins/services/src/js/pages/task-details/TaskLogsTab.js [hunk 1]
```diff
@@ -234,22 +234,21 @@ class TaskLogsTab extends mixin(StoreMixin) {
   getDownloadButton() {
     const {task} = this.props;
     const {selectedStream} = this.state;
-    const params = getLogParameters(task, {filter: {STREAM: selectedStream}});
+    const params = getLogParameters(task, {
+      postfix: selectedStream.toLowerCase(), // This will be added to the name
```
Comment (jfurrow) [inline plugins/services/src/js/pages/task-details/TaskLogsTab.js]: It looks like `selectedStream` may be undefined here

Comment (mlunoe) [inline plugins/services/src/js/pages/task-details/TaskLogsTab.js]: Yup, that's the problem. Thanks for noticing. That's what you get for making a minor change right at the end and don't test it through :man_shrugging:

Comment (mlunoe) [inline plugins/services/src/js/pages/task-details/TaskLogsTab.js]: Fixed

**Comment by mlunoe** [comment]
Fixed!

**Comment by jfurrow** [comment]
Looks good!

**Comment by mlunoe** [comment]
Rebased with integration test fixes

**Comment by mlunoe** [comment]
Restest this please

**Comment by rcorral** [comment]
retest this please

**Comment by rcorral** [comment]
retest this please

**Pull request merged**